cmake/build_common.cmake

```diff
@@ -0,0 +1,192 @@
+if(NOT DEFINED ROOT OR NOT DEFINED ARCH)
+    message(
+        FATAL_ERROR 
+        "Assert: ROOT = ${ROOT}; ARCH = ${ARCH}"
+    )
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/Version.cmake")
+
+set(
+    BUILD
+    1
```

Comment:
I think it should be 0, and when you want to specify build_number you will set it in Enviroment variables. (The same behavior with TAG)

Reply:
Ok, replaced with 0. Thanks!